pkg/scaffold/external_templates.go

```diff
@@ -318,7 +320,7 @@ func newTemplateNotFoundError(templateDir string, templateName string) error {
 	// Attempt to read the directory to offer suggestions.
 	infos, err := ioutil.ReadDir(templateDir)
 	if err != nil {
-		contract.IgnoreError(err)
+		log.Infof("ioutil.ReadDir(%s) error: %v", templateDir, err)
```

Comment:
Consider using `Warnf` output for ignored errors

Reply:
ok, thanks for tips